### PR TITLE
Setup github actions to release django-khronos package to PyPi. 

### PR DESCRIPTION
**Pull request by Vivy24**
Done #6 
I have set up the action github to publish the package into PyPi. 

The last step , you have to do is configure your PyPi account as step 1 "Configuring trusted publishing" from this link: https://packaging.python.org/en/latest/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/ with the project name is "Khronos-Py".  I do not want to do this step for you because this project is belong to you and I should not be the owner on PyPi. Then the script should work. Let me know if you have any more question or error

### CODE REVIEW

**Comment by Vivy24** [comment]
Hi @abhinavsp0730 , could you take a look at the PR ? Thank you and have a great day

File: Khronos/setup.py
Comment (abhinavsp0730) [inline Khronos/setup.py:1]: Why this? See this PR https://github.com/abhinavsp0730/khronos/pull/9

File: Khronos/README.md
Comment (abhinavsp0730) [inline Khronos/README.md:1]: Would you please remove this? There should be one `README.MD` file which is present in the root dir?

File: README.md
Comment (abhinavsp0730) [inline README.md:1]: Remove this. If this PR got merged this will automatically gonna update.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -11,4 +11,41 @@ jobs:
...
+                name: python-package-distributions
+                path: Khronos/dist/
+    publish-to-pypi:
+        name: Publish Khronos to PyPi
+        needs:
+            - build-project
+        runs-on: ubuntu-latest
+        environment:
+            name: pypi
+            url: https://pypi.org/p/Khronos-Py
```
Comment (abhinavsp0730) [inline .github/workflows/main.yml:40]: use this https://pypi.org/project/django-khronos/

**Comment by abhinavsp0730** [comment]
Left comments. Other than that LGTM. Thanks :)

**Pull request closed without merging**